tutorials/Class_Balance.ipynb

```diff
@@ -163,12 +163,12 @@
     "Now we can directly generate $O$.\n",
     "By our conditional independence assumption, for distinct $i,j,k$, we have:\n",
     "$$\n",
-    "P(\\lambda_i = y', \\lambda_j = y'', \\lambda_k = y''' | Y = y) = C_{i,y',y} C_{j,y'',y} C_{j,y''',y}\n",
+    "P(\\lambda_i = y', \\lambda_j = y'', \\lambda_k = y''' | Y = y) = C_{i,y',y} C_{j,y'',y} C_{k,y''',y}\n",
     "$$\n",
     "\n",
     "Thus we have:\n",
     "$$\n",
-    "O_{i,j,k,y',y'',y'''} = \\sum_y P(Y=y) C_{i,y',y} C_{j,y'',y} C_{j,y''',y}\n",
+    "O_{i,j,k,y',y'',y'''} = \\sum_y P(Y=y) C_{i,y',y} C_{j,y'',y} C_{k,y''',y}\n",
     "$$\n",
```

Comment:
@ajratner can you confirm this (and the one above) is the correct change?

Reply:
Yup!  @chmccreery good catch!